pages/ourteam.js

```diff
@@ -64,8 +64,8 @@ class Ourteam extends Component {
               <h1>Our Team</h1>
               <div>
                 <p>
-                  We are a team of five University of Victoria Engineering students.
-                  Brakes Supreme was created as part of our capstone project course: ECE/SENG 499.
+                  We are a team of six University of Victoria Engineering students who formed
```

Comment:
good fix here, just noticed it now in the screenshot and was about to comment on it but then I saw your fix 👍